### PR TITLE
add coin info for markets

### PR DESCRIPTION
**Pull request by Pantani**
closes https://github.com/trustwallet/blockatlas/issues/682

`http://localhost:8420/v1/market/info?currency=USD&coin=60`

```
{
  "volume_24": 10408694219.2302,
  "market_cap": 17398597896,
  "circulating_supply": 109405422,
  "total_supply": 109405422,
  "info": {
    "name": "Ethereum",
    "website": "https://ethereum.org/",
    "source_code": "https://github.com/ethereum",
    "whitepaper": "https://github.com/ethereum/wiki/wiki/White-Paper",
    "explorers": [
      {
        "name": "Ethereum",
        "url": "https://etherscan.io/"
      }
    ],
    "socials": [
      {
        "name": "Twitter",
        "url": "https://twitter.com/ethereum"
      },
      {
        "name": "Reddit",
        "url": "https://www.reddit.com/r/ethereum/"
      }
    ],
    "details": [
      {
        "language": "en",
        "description": "Open source platform to write and distribute decentralized applications."
      }
    ],
    "data_source": "crowd"
  }
}
```

### CODE REVIEW

**Comment by vikmeup** [comment]
`v1/market/info?currency=USD&coin=60?token=0x6b175474e89094c44da98b954eedeac495271d0f` this endpoint does not work

it needs to support both coins and tokens

File: services/assets/client.go [hunk 1]
```diff
@@ -12,6 +12,16 @@ const (
 	AssetsURL = "https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/"
 )
 
+func GetCoinInfo(coinId int) (info blockatlas.CoinInfo, err error) {
```
Comment (vikmeup) [inline services/assets/client.go]: this needs to take `token: String`

Data would be located here: https://github.com/trustwallet/assets/tree/master/blockchains/ethereum/assets/0x6B175474E89094C44Da98b954EedeAC495271d0F

Comment (vikmeup) [reply services/assets/client.go]: Once we add it.

Comment (Pantani) [reply services/assets/client.go]: can we use:
`https://github.com/trustwallet/assets/tree/master/blockchains/ethereum/assets/0x6B175474E89094C44Da98b954EedeAC495271d0F/info/info.json`

instead
`https://github.com/trustwallet/assets/tree/master/blockchains/ethereum/assets/0x6B175474E89094C44Da98b954EedeAC495271d0F/info.json`

?

Comment (vikmeup) [reply services/assets/client.go]: I would rather move info.json out of info folder instead on assets repo, for consistency, otherwise too many folders

**Pull request merged**